app/build.gradle

```diff
@@ -49,9 +49,9 @@ android {
     }
 
     applicationVariants.all { variant ->
-        variant.outputs.each { output ->
-            output.outputFile = new File(
-                    output.outputFile.parent, "GnucashAndroid_v${variant.versionName}.apk")
+        variant.outputs.all { output ->
+            //output.outputFile = new File(output.outputFile.parent, "GnucashAndroid_v${variant.versionName}.apk")
```

Comment:
Unless there's some good reason to leave the old line commented, please, remove it. Git already takes care of remembering it.